Solutions/Rapid7InsightVM/Data Connectors/InsightVMCloudAPISentinelConnector/__init__.py

```diff
@@ -46,12 +45,21 @@ async def main(mytimer: func.TimerRequest):
     logging.info('Script started.')
     async with aiohttp.ClientSession() as session_api:
         async with aiohttp.ClientSession() as session_sentinel:
+            delay = os.environ.get('Delay', "60")

```

Comment:
@KateShapovalova : Since you are reading the value for **Delay** and **ShiftStartTime** from configuration that needs to be added in the [azure deploy ](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Rapid7InsightVM/Data%20Connectors/azuredeploy_Connector_InsightVMCloudAPI_AzureFunction.json)file as well. Please update it.
Thanks!!

Reply:
@v-rucdu done.